v3/deprecated/20241205-v3-stable-pool/test/task.fork.ts

```diff
@@ -7,7 +7,7 @@ import { ONES_BYTES32, ZERO_ADDRESS } from '@helpers/constants';
 import { bn, fp } from '@helpers/numbers';
 import { StablePoolDeployment } from '../input';
 
-describeForkTest('StablePool-V3', 'mainnet', 21336200, function () {
+describeForkTest.skip('StablePool-V3', 'mainnet', 21336200, function () {
```

Comment:
Nice to know that we need to actively skip the tests inside the deprecated folder. I wonder if there's a way to automatically ignore these tests.